client/blocks/login/index.jsx

```diff
@@ -191,7 +195,7 @@ class Login extends Component {
 		}
 
 		return (
-			<LoginForm onSuccess={ this.handleValidUsernamePassword } />
+			<LoginForm onSuccess={ this.handleValidUsernamePassword } privateSite={ privateSite } />
```

Comment:
I'm not a great fan of the `privateSite` name as you could rightfully assume that this prop contains a `Site` object whereas it's just a boolean. Hence I suggest renaming it into `isPrivateSite`).